DependencyInjection/Configuration.php

```diff
@@ -428,22 +428,31 @@ private function addModelSection(ArrayNodeDefinition $node)
     /**
      * @param ArrayNodeDefinition $node
      */
-    private function addBuzzSection(ArrayNodeDefinition $node)
+    private function addHttpClientSection(ArrayNodeDefinition $node)
     {
         $node
             ->children()
-                ->arrayNode('buzz')
+                ->arrayNode('http')
                     ->addDefaultsIfNotSet()
                     ->children()
-                        ->scalarNode('connector')->defaultValue('sonata.media.buzz.connector.curl')->end()
                         ->arrayNode('client')
                         ->addDefaultsIfNotSet()
                         ->children()
-                            ->booleanNode('ignore_errors')->defaultValue(true)->end()
-                            ->scalarNode('max_redirects')->defaultValue(5)->end()
+                            ->booleanNode('http_errors')->defaultValue(false)->end()
+                            ->arrayNode('allow_redirects')
```

Comment:
and it should be possible to also configure the message factory, defaulting to `httplug.message_factory.default`. you need that in the places where you want to send requests, as the client is no longer a factory for messages.